ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/block/AbstractBlockProcessor.java

```diff
@@ -838,14 +839,27 @@ protected BlockValidationResult verifyVoluntaryExits(
     for (SignedVoluntaryExit signedExit : exits) {
       boolean exitSignatureValid =
           operationSignatureVerifier.verifyVoluntaryExitSignature(
-              state.getFork(), state, signedExit, signatureVerifier);
+              state,
+              signedExit,
+              signatureVerifier,
+              computeVoluntaryExitDomain(
```

Comment:
think it's more readable to separate this in a variable inside the loop

Reply:
Done. In `Spec::verifyVoluntaryExitSignature` too